app/models/role_loader.rb

```diff
@@ -0,0 +1,45 @@
+class RoleLoader
+
+  def available_roles_for_application(name)
```

Comment:
Yes we do.
When a User is logging in the provider checks that the User holds at least 1 role that the Application supports (otherwise no access_token is issued).
In this case it doesn't matter which Organisation the User belongs to, just that they have one of the available roles